### ISSUE TITLE
Update SwaggerUI

### ISSUE THREAD

**Issue by martinszy** [phil-mitchell/exegesis-plugin-swagger-ui-express#21]
NPM is informing of a vulrability in this package's dependencies.

```
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ Moderate      │ Server side request forgery in SwaggerUI                     │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ swagger-ui-dist                                              │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Patched in    │ >=4.1.3                                                      │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ exegesis-plugin-swagger-ui-express                           │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ exegesis-plugin-swagger-ui-express > swagger-ui-express >    │
│               │ swagger-ui-dist                                              │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://github.com/advisories/GHSA-qrmm-w75w-3wpx            │
└───────────────┴──────────────────────────────────────────────────────────────┘

```

**Comment by phil-mitchell** [comment]
Fixed by #22

**Comment by martinszy** [comment]
Thank you Phil, much appreciated :)
